src/window.rs

```diff
@@ -316,6 +316,18 @@ impl Window {
     pub fn set_max_dimensions(&self, dimensions: Option<(u32, u32)>) {
         self.window.set_max_dimensions(dimensions)
     }
+    
+    /*
+    /// Sets whether the window is resizable or not.
+    ///
+    /// ## Platform-specific
+    ///
+    /// This only has an effect on Windows, X11, and macOS.
+    */
```

Comment:
You left the block comment open/close in.

Reply:
Heh, good catch. I'm not sure I would have ever seen that as a code reviewer.